docs/framework/winforms/automatic-scaling-in-windows-forms.md

```diff
@@ -47,12 +47,20 @@ Automatic scaling enables a form and its controls, designed on one machine with
   
  While this mechanism was sufficient for most purposes, it suffered from the following limitations:  
   
--   Since the <xref:System.Windows.Forms.Form.AutoScaleBaseSize%2A> property represents the baseline font size as integral values, rounding errors occur that become evident when a form is cycled through multiple resolutions.  
+-   Since the <xref:System.Windows.Forms.Form.AutoScaleBaseSize%2A> property represents the baseline font size as integer values, rounding errors occur that become evident when a form is cycled through multiple resolutions.  
   
 -   Automatic scaling was implemented in only the <xref:System.Windows.Forms.Form> class, not in the <xref:System.Windows.Forms.ContainerControl> class. As a result, user controls would scale correctly only when the user control was designed at the same resolution as the form, and it was placed in the form at design time.  
   
--   Forms and their child controls could only be concurrently designed by multiple developers if their machine resolutions were the same. Likewise it also made inheritance of a form dependent on the resolution associated with the parent form.  
-  
+-   Forms and their child controls could only be concurrently designed by multiple developers if their machine resolutions were the same. Likewise it also made inheritance of a form dependent on the resolution associated with the parent form.
+
+> [!NOTE]
+> With display DPIs, especially in modern 2-in-1 devices, differing extremely, this can still happen with the most current versions of the .NET Framework and Visual Studio. To address this in a team using different DPI displays, make sure Visual Studio always starts in a non-High-DPI-aware mode, so the Windows Forms designer always bases the layout calculation on 96 DPI. To this end, simply set the following regestry key to disable Visual Studio's HighDPI awareness:
+
+```
+[HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Image File Execution Options\devenv.exe]
+"dpiAwareness"=dword:00000000
+```
```

Comment:
@Tanya-Solyanik I haven't added the link. Wondering if that's something that should be documented in our own docs instead of going to an external blog.